### PR TITLE
Feature#3. Create SignUp form.

### PR DESCRIPTION
**Pull request by yovanha**
# Description
 - We've created a TextIput component for the Name, Email and Password fields.
 - We've created a CheckBox component for the Terms and Privacy Policy.
 - We've created a Button component for the SignUp and SignUp with Google.
 - We've added these components to the SignUp screen.
 
  ## Screenshots 
image

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -1,8 +1,9 @@
 import React from 'react';
-import {Home, Login} from './src/containers';
+import { StyleSheet, TextInput} from 'react-native';
+import {Home, Login, SignUp} from './src/containers';
 
 const App = () => {
-  return <Login/>;
+  return <SignUp/>
```
Comment (cfcortes) [inline App.js:6]: Codacy has a fix for the issue: [Replace `/>` with `·/>;`](https://app.codacy.com/gh/BC-MAY-21-RN/proyecto-proy-t01/pullRequest?prid=7869790)
```suggestion
  return <SignUp />;
```

File: src/containers/SignUp/index.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+            <Formik>
+            <View>
+               <View>
+               <TextInputField label='Nombre'/>
+               </View>
+                <View>
+                   <TextInputField label='Correo'/>
+                </View>
+                <View>
+                   <TextInputField label='Contraseña'/>
```
Comment (cfcortes) [inline src/containers/SignUp/index.js:19]: Codacy has a fix for the issue: [Unexpected usage of singlequote.](https://app.codacy.com/gh/BC-MAY-21-RN/proyecto-proy-t01/pullRequest?prid=7869790)
```suggestion
                   <TextInputField label="Contraseña"/>
```

File: src/containers/SignUp/index.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+import {Formik} from 'formik';
+import { CheckBoxField, TextInputField, CustomButton } from '../../components';
+
+const SignUp = () => {
+    return (
+        <SafeAreaView>
+            <Formik>
+            <View>
+               <View>
+               <TextInputField label='Nombre'/>
```
Comment (cfcortes) [inline src/containers/SignUp/index.js:13]: Codacy has a fix for the issue: [Unexpected usage of singlequote.](https://app.codacy.com/gh/BC-MAY-21-RN/proyecto-proy-t01/pullRequest?prid=7869790)
```suggestion
               <TextInputField label="Nombre"/>
```

File: src/containers/SignUp/index.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+               </View>
+                <View>
+                   <TextInputField label='Correo'/>
+                </View>
+                <View>
+                   <TextInputField label='Contraseña'/>
+                </View>
+                <View>
+                   <CheckBoxField label='Acepto los Terminos y Condiciones.'/>
```
Comment (cfcortes) [inline src/containers/SignUp/index.js:22]: Codacy has a fix for the issue: [Unexpected usage of singlequote.](https://app.codacy.com/gh/BC-MAY-21-RN/proyecto-proy-t01/pullRequest?prid=7869790)
```suggestion
                   <CheckBoxField label="Acepto los Terminos y Condiciones."/>
```

File: src/containers/SignUp/index.js [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+                </View>
+                <View>
+                   <CustomButton text='Registrarme'/>
+                </View>
+                <View>
+                   <CustomButton text='Registrarme con Google'/>
```
Comment (cfcortes) [inline src/containers/SignUp/index.js:28]: Codacy has a fix for the issue: [Unexpected usage of singlequote.](https://app.codacy.com/gh/BC-MAY-21-RN/proyecto-proy-t01/pullRequest?prid=7869790)
```suggestion
                   <CustomButton text="Registrarme con Google"/>
```

File: src/containers/SignUp/index.js [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+                   <TextInputField label='Correo'/>
+                </View>
+                <View>
+                   <TextInputField label='Contraseña'/>
+                </View>
+                <View>
+                   <CheckBoxField label='Acepto los Terminos y Condiciones.'/>
+                </View>
+                <View>
+                   <CustomButton text='Registrarme'/>
```
Comment (cfcortes) [inline src/containers/SignUp/index.js:25]: Codacy has a fix for the issue: [Unexpected usage of singlequote.](https://app.codacy.com/gh/BC-MAY-21-RN/proyecto-proy-t01/pullRequest?prid=7869790)
```suggestion
                   <CustomButton text="Registrarme"/>
```

File: App.js [hunk 7]
```diff
@@ -1,8 +1,9 @@
 import React from 'react';
-import {Home, Login} from './src/containers';
+import { StyleSheet, TextInput} from 'react-native';
```
Comment (cfcortes) [inline App.js:2]: Codacy has a fix for the issue: [Delete `·`](https://app.codacy.com/gh/BC-MAY-21-RN/proyecto-proy-t01/pullRequest?prid=7869790)
```suggestion
import {StyleSheet, TextInput} from 'react-native';
```

File: src/containers/SignUp/index.js [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+const SignUp = () => {
+    return (
+        <SafeAreaView>
+            <Formik>
+            <View>
+               <View>
+               <TextInputField label='Nombre'/>
+               </View>
+                <View>
+                   <TextInputField label='Correo'/>
```
Comment (cfcortes) [inline src/containers/SignUp/index.js:16]: Codacy has a fix for the issue: [Unexpected usage of singlequote.](https://app.codacy.com/gh/BC-MAY-21-RN/proyecto-proy-t01/pullRequest?prid=7869790)
```suggestion
                   <TextInputField label="Correo"/>
```

**Review by DemisRincon — APPROVED** [review]

**Review by DemisRincon — APPROVED** [review]

**Review by cortezroberto — APPROVED** [review]
LGTM.

**Review by hancel-avila — APPROVED** [review]
Well done! 👍🏻

**Pull request merged**